app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variable = {
+    "mutable": [],
+    "immutable": [],
+}
+
+
+def determine_type_variable(var) -> dict:
+
+    if type(var) in [set, dict, list]:
+        sorted_variable["mutable"].append(var)
+    else:
+        sorted_variable["immutable"].append(var)
+
+    return sorted_variable
```

Comment:
<img width="293" alt="image" src="https://user-images.githubusercontent.com/72562232/184360716-b80ad0c7-ddf3-4385-9602-c20ad7c7405f.png">
You shouldn't create a function for this action, you should manual write all variables to list